### ISSUE TITLE
Fix text contrast issues, particularly in dark mode

### ISSUE THREAD

**Issue by chriszs** [BadIdeaFactory/bad-decision-maker#26]

**Comment by chriszs** [comment]
According to WCAG guidelines, these BIF brand colors are okay for use as large text colors on dark backgrounds:

- #009c9b
- #31a936
- #9db41f
- #f39200
- #e94e1b

These BIF brand colors are okay for use as large text colors on light backgrounds:

- #b3115a
- #af1781
- #883a8e
- #8f0863
- #54358c
- #625198
- #312783
- #24378d
- #224c9c
- #1d71b8
- #e30613
- #be1622

**Comment by chriszs** [comment]
The inverse should also be true: large dark text should be okay on the first set, large white text on the second set.
